lib/dor/services/client/workspace.rb

```diff
@@ -24,13 +24,21 @@ def create(source:)
         end
 
         # Cleans up a workspace
-        # @raise [UnexpectedResponse] if the request is unsuccessful.
-        # @return nil
-        def cleanup
+        # @raise [NotFoundResponse] when the response is a 404 (object not found)
+        # @raise [UnexpectedResponse] when the response is not successful.
+        # @param [String] workflow (nil) which workflow to callback to.
+        # @param [String] lane_id for prioritization (default or low)
+        # @return [String] the URL of the background job on dor-service-app
+        def cleanup(workflow: nil, lane_id: nil)
```

Comment:
This is merged, so this is more of a question that a request to change something. Wondering if you considered using `.compact!` on a hash of the workflow / lane params and then, since active_support is available, `to_query` to construct the params? Would save some string wrangling. Or is this a pattern we use in DorServicesClient to construct requests?

Reply:
That is the pattern. However, I have to admit that when I was re-using it, it seemed quite inelegant to me. Your suggestion seems much cleaner.